packages/amplify-e2e-tests/src/rds-v2-tests-common/rds-auth-userpool-static-dynamic-fields.ts

```diff
@@ -0,0 +1,610 @@
+import {
```

Comment:
The tests are (rightly) the source of truth for validating that our system conforms to the expected behavior, but it’s really hard to read them to derive that behavior. How can we summarize the expected behaviors in human readable form for easy reference later? For example, could we restructure the test to do all the setup in a describe, and then make each test label clearly describe the behaviors you’re currently noting in a comment?

Reply:
E.g., `// notes, team and salary are protected and cannot be read upon mutation` would become `test('notes, team and salary are protected and cannot be read upon mutation' ...)`